build/lib/apple_health_exporter/export.py

```diff
@@ -0,0 +1,63 @@
+import argparse
```

Comment:
`build` and `dist` are included in `.gitignore`, but seem to have accidentally been included in the PR files. Could you remove these folders from the PR, since they should only be generated at build time?